File: server/project/mj-client/src/main/java/majiang/client/portal/admin/model/SettingModel.java
package majiang.client.portal.admin.model;

import org.forkjoin.apikit.core.Message;

/**
 * @author zuoge85@gmail.com on 2017/1/11.
 */
@Message
public class SettingModel {
    /**广告（首页）*/
    private String notice;
    /**广播（跑马灯）*/
    private String radio;
    /**充值信息*/
    private String payInfo;
    /**用户协议*/
    private String agreement;
    /**用户默认房卡*/
    private int initGold;

    public String getNotice() {
        return notice;
    }

    public void setNotice(String notice) {
        this.notice = notice;
    }

    public String getRadio() {
        return radio;
    }

    public void setRadio(String radio) {
        this.radio = radio;
    }

    public String getPayInfo() {
        return payInfo;
    }

    public void setPayInfo(String payInfo) {
        this.payInfo = payInfo;
    }

    public String getAgreement() {
        return agreement;
    }

    public void setAgreement(String agreement) {
        this.agreement = agreement;
    }

    public int getInitGold() {
        return initGold;
    }

    public void setInitGold(int initGold) {
        this.initGold = initGold;
    }

    @Override
    public String toString() {
        return "SettingModel{" +
                "notice='" + notice + '\'' +
                ", radio='" + radio + '\'' +
                ", payInfo='" + payInfo + '\'' +
                ", agreement='" + agreement + '\'' +
                ", initGold=" + initGold +
                '}';
    }
}

File: server/manager/agent-js-sdk/model/SettingModel.js
'use strict';



/**
 * @author  zuoge85@gmail.com on 2017/1/11.
*/
class SettingModel {

	/**
	 * 广告（首页）
	 */
    notice:String;

	/**
	 * 广播（跑马灯）
	 */
    radio:String;

	/**
	 * 充值信息
	 */
    payInfo:String;

	/**
	 * 用户协议
	 */
    agreement:String;

	/**
	 * 用户默认房卡
	 */
    initGold:Number;
    constructor() {

    }

    formObject({notice,radio,payInfo,agreement,initGold}):SettingModel{
        this.notice = notice;
        this.radio = radio;
        this.payInfo = payInfo;
        this.agreement = agreement;
        this.initGold = initGold;
        return this;
    }

    static of({notice,radio,payInfo,agreement,initGold}):SettingModel{
        return new SettingModel().formObject({notice,radio,payInfo,agreement,initGold});
    }

    static form(notice:String,radio:String,payInfo:String,agreement:String,initGold:Number):SettingModel{
        return new SettingModel().formObject({notice,radio,payInfo,agreement,initGold});
    }
}

export default SettingModel;